src/main/java/org/apache/commons/text/similarity/SorensenDiceSimilarity.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.text.similarity;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Function;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * A similarity algorithm indicating the percentage of matched characters
+ * between two character sequences.
+ *
+ * <p>
+ * The S&#248;rensen–Dice coefficient is a statistic used for comparing the
```

Comment:
The documentation is correctly rendered now on GitHub, but not on Eclipse and I believe the issue is with the _dash_ character you are using @ameyjadiye . Take a look at this screenshot:

![image](https://user-images.githubusercontent.com/304786/54877379-9724d800-4e82-11e9-9e09-2d0916fd507e.png)

Now if you copy the character between Sorensen and Dice, and paste here: https://unicode-table.com/en/, it should tell you it is a `En dash`, but probably Javadocs behave better with a hyphen-minus instead: https://unicode-table.com/en/search/?q=-

Reply:
well, this wasn't the problem with my eclipse though. can we have any rule in Javadoc plugin that only standard English characters must be used else build should fail? corrected here btw.